desktop/mac/release-notes/index.md

```diff
@@ -19,14 +19,46 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
-## Docker Desktop 4.2.0
-2021-11-09
+## Docker Desktop 4.3.0
+2021-12-02
 
 > Download Docker Desktop
 >
 > [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
 > [Mac with Apple chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn }
 
+### Upgrades
+
+- [Docker Engine v20.10.11](https://docs.docker.com/engine/release-notes/#201011)
+- [containerd v1.4.12](https://github.com/containerd/containerd/releases/tag/v1.4.12)
+- [Buildx 0.7.1](https://github.com/docker/buildx/releases/tag/v0.7.1)
+- [Compose v2.2.1](https://github.com/docker/compose/releases/tag/v2.2.1)
```

Comment:
I think we may want to elaborate a bit for the components we own (Engine, Buildx, Compose, …) with a sub-list of changes? Or is this something we’ve decided not to do in the end @usha-mandya?

Reply:
I don't think it was intentional. We can add a list highlighting some important updates as per https://docs.docker.com/desktop/mac/release-notes/#docker-desktop-401.